Gruntfile.js

```diff
@@ -474,6 +474,25 @@ module.exports = function(grunt) {
                 reloadOnRestart: true,
                 watchTask: true
             }
+        },
+
+        validation: {
+            options: {
+                reset: grunt.option('reset') || false,
+                stoponerror: false,
+                maxTry: 3,
+                relaxerror: ['Bad value X-UA-Compatible for attribute http-equiv on element meta.'], //ignores these errors 
+                generateReport: true,
+                errorHTMLRootDir: "w3cErrorFolder",
+                useTimeStamp: true,
+                errorTemplate: "w3cErrorFolder/w3c_validation_error_Template.html"
```

Comment:
This might be more appropriate to put in the `/tests` directory, `/tests/validation` perhaps?